proto/google/api/http.proto

```
@@ -0,0 +1,370 @@
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api;
```

Buf (1.47.2)

Files with package "google.api" must be within a directory "google/api" relative to root but were in directory "proto/google/api".

(PACKAGE_DIRECTORY_MATCH)